src/main/java/duke/AddCommand.java

```diff
@@ -0,0 +1,47 @@
+package duke;
+
+/**
+ * AddCommand extends command and adds to the taskList based on the input
+ */
+public class AddCommand extends Command {
+
+    AddCommand(String input){
+        super(input);
+    }
+
+    /**
+     * Executes the add command based on the task type. Adds accordingly.
+     * @param tasks from Tasklist initialised from duke.
+     * @param ui from UI initialised from duke.
+     * @param storage from storage initialised from duke.
+     * @throws DukeException might throw DukeException from saveData.
+     */
+    protected void execute(TaskList tasks, Ui ui, Storage storage) throws DukeException {
+        String[] strArr = input.split(" ");
+        String type = strArr[0];
+        switch (type) {
+            case "deadline":
```

Comment:
I believe in the CS2103 coding standards, the block of case codes should be at the same indentation as the switch comments. You can change this indentation setting in IntelliJ automatically.